server/pages/index.jade

```diff
@@ -11,10 +11,8 @@ doctype html
 
 html(lang=lang, dir=isRTL ? 'rtl' : 'ltr', class=isFluidWidth ? 'is-fluid-width' : '')
 	head
-		if helmetTitle
```

Comment:
For non-iso, it's only `documentHead`. For iso, it's unchanged (i.e. `documentHead`, `ui`, and `themes`).